### DISCUSSION TITLE
Phalcon Datatables

### DISCUSSION THREAD

**Discussion by mis-uep** [phalcon/cphalcon#15827] [Q&A]
Hi guys! Do you have experience in using datatables with server-side processing? I'm trying m1ome datatables but I can't make it work.

**Comment by gnikolopoulos** [comment]
I have used datatables quite a few times within my Phalcon apps and I think the best solution is to create your own controller action to handle the AJAX request. I tried m1ome in the beginning as well but I never got it to make it work as I wanted it, plus is was way to slow for me.
